swarm/network/simulation/node.go

```diff
@@ -121,6 +128,8 @@ func (s *Simulation) AddNode(opts ...AddNodeOption) (id enode.ID, err error) {
 	if err != nil {
 		return id, err
 	}
+	s.buckets[node.ID()] = new(sync.Map)
```

Comment:
- If I call `DownNodeIDs()` twice, does that make sense to override the value with a new "sync.Map" at the second call?
- Why do we store in `buckets` in `DownNodeIDs()` but not in `UpNodeIDs()`?

Reply:
I don't undesrtand this comment. We are in `AddNode` here

Reply:
My mistake! Excuse me. Ignore pls. :+1: